app/views/analyses/show.html.erb

```diff
@@ -81,7 +77,9 @@
     </div>
 
   <div class="l-analysis-advise">
-    <a href="#"><p class="text -link -medium -is-not-printable">Learn more about our methodology</p></a>
+    <%= link_to "/documents/Grow Asia Counter Methodology.pdf", target: "_blank" do %>
```

Comment:
It's not in the database. It's in a folder inside the project. We could remove the spaces, but it shouldn't really cause any issues.